static/sass/_patterns_card.scss

```diff
@@ -35,6 +35,8 @@
   %post-card-header {
     border-radius: 2px;
     padding: $sp-small $sp-medium;
+    margin-bottom: 0;
+    border-bottom: 0;
```

Comment:
If you were removing the margin and border from a set of cards. You may want to consider using the Matrix pattern instead: https://docs.vanillaframework.io/en/patterns/matrix